tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -186,8 +191,12 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_stuffer_wipe(&client_conn->in));
         EXPECT_SUCCESS(s2n_stuffer_wipe(&client_conn->out));
 
+        EXPECT_SUCCESS(s2n_hmac_digest_size(server_conn->server->server_record_mac.alg, &mac_digest_size));
+        expected_encrypted_bytes_out = server_conn->encrypted_bytes_out + mac_digest_size +  TLS13_CONTENT_TYPE_LENGTH + 
+            server_conn->server->cipher_suite->record_alg->cipher->io.aead.tag_size + deadbeef_from_server.size;
```

Comment:
Why set expected_encrypted_bytes_out to something this specific? Why not just 0?